homeassistant/components/intellifire/manifest.json

```diff
@@ -7,5 +7,7 @@
   "dependencies": [],
   "codeowners": ["@jeeftor"],
   "iot_class": "local_polling",
-  "loggers": ["intellifire4py"]
+  "loggers": ["intellifire4py"],
+  "dhcp": [{"hostname": "zentrios-*"}]
```

Comment:
This matcher looks a bit generic https://docs.zentri.com/zentrios/wz/3.6/

Reply:
This is supposed to be a draft review :) ... but we can talk about it regardless.

I don't have specific information from the manufacture on any mac address matching - mostly - the thread listed above some folks in the community posted their host name... and i then searched to make sure there wasnt any other Zentrios mentioned anywhere else in HA.  

The only other thing I can do is go back and ask people to post their mac addresses - but I wasn't sure if they would be happy to do so.

Reply:
I'd expect they are all starting with
https://ouilookup.com/vendor/zentri-pty-ltd

Reply:
Unfortunately a discovery in this case likely means they just selected zentri as their IoT vendor, and its not likely to be a fireplace since other vendors use their tech.

That is probably ok as long as the probing isn't too cpu intensive or expensive.